We neither support Java 1.6 nor need these classes to use ecj.

--- origsrc/build.xml	2007-02-23 15:17:54.000000000 -0600
+++ src/build.xml	2009-11-29 00:40:35.061069800 -0600
@@ -17,17 +18,6 @@
 			<compilerarg line="-Xlint:none"/>
 		</javac>
 
-		<javac destdir="${output}"
-				debuglevel="lines,source"
-				source="1.6"
-				target="1.6"
-				includes="org/eclipse/jdt/internal/compiler/tool/*">
-			<src path="${basedir}"/>
-			<include name="org/eclipse/jdt/internal/compiler/tool/*"/>
-			<include name="**/apt/**"/>
-			<compilerarg line="-Xlint:none"/>
-		</javac>
-
 		<zip destfile="${basedir}/${jar_file}">
 			<fileset dir="${output}" />
 			<fileset dir="${basedir}">
